static/app/views/replays/detail/layout/chooseLayout.tsx

```diff
@@ -3,6 +3,7 @@ import styled from '@emotion/styled';
 import DropdownControl, {DropdownItem} from 'sentry/components/dropdownControl';
 import {IconPanel} from 'sentry/icons';
 import useUrlParam from 'sentry/utils/replays/hooks/useUrlParams';
+import {handleDefaultLayout} from 'sentry/views/replays/detail/layout/utils';
```

Comment:
I would rather call this `getDefaultLayout` since it's not an event handler just for clarity